website/tests/pages/submission/index.spec.ts

```diff
@@ -48,16 +48,7 @@ test.describe('The submit page', () => {
         await submitPage.confirmationINSDCTerms.click();
         await submitPage.confirmationNoPII.click();
         await submitPage.submitButton.click();
-        await expect(submitPage.page.getByText('Response Sequence Headers')).toBeVisible();
-
-        const responseHeaderLocator = submitPage.page.getByText(
-            `${restrictedDataUseTermsType} until: ${dateTimeInMonths(6).toFormat('yyyy-MM-dd')}`,
-            {
-                exact: false,
-            },
-        );
-        expect(await responseHeaderLocator.count()).toBe(testSequenceCount);
 
-        // TODO(#702): Redirect to the review page after submission is successful, and check the data use terms there
+        await submitPage.page.waitForURL(`${baseUrl}${routes.userSequenceReviewPage(dummyOrganism.key, groupId)}`);
```

Comment:
So this tests that you can go through the UI for submitting restricted data but it doesn't actually test that the UI actually correctly captures that choice and sends it to the server. But maybe that's too hard atm. We should try to full E2E soon which would let us do that.

Reply:
Exactly. It is not possible with our current e2e test set up to test the full process. The user sets a sequence to state restricted before we send it to the prepro pipeline and the e2e tests mock the entire interaction with the prepro pipeline. So in theory the prepro pipeline could just remove the restricted label and send us back open data and this test would not detect it.